cime_config/machines/config_machines.xml

```diff
@@ -1036,7 +1036,7 @@
     <OS>CNL</OS>
     <COMPILERS>crayclang-scream</COMPILERS>
     <MPILIBS>mpich</MPILIBS>
```

Comment:
This is the only file that I am not sure all the necessary changes are in correctly as it is the only place that had major DIFFs between the cess-production branch and new master changes.

At @ambrad and @oksanaguba, I see you two on the git blame. Could you take a look at the `<machine MACH="frontier-scream-gpu">` entry to make sure we have what we need (specifically around the ENV changes below).

Reply:
Conrad, did your editor make all the whitespace changes? If so, I recommend disabling that auto-whitespace-fix option to limit the changes made in this file. (I can filter these out on github's view, of course, but the more important issue is to minimize the footprint of the changes.)

Once you do that, my understanding is this file is supposed to look exactly like the master branch's version.

Reply:
OK, updated to match master and remove whitespaces.